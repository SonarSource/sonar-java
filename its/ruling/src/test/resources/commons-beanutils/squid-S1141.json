{
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils2/ConstructorUtils.java':[
373,
],
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils2/MappedPropertyDescriptor.java':[
102,
],
}
