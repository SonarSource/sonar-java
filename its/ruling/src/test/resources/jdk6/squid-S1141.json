{
'jdk6:java/awt/EventDispatchThread.java':[
374,
],
'jdk6:java/awt/Toolkit.java':[
833,
838,
843,
846,
],
'jdk6:java/awt/font/StyledParagraph.java':[
418,
],
'jdk6:java/net/Inet6Address.java':[
405,
],
'jdk6:java/net/InetAddress.java':[
1356,
],
'jdk6:java/net/PlainSocketImpl.java':[
145,
],
'jdk6:java/net/URL.java':[
1139,
],
'jdk6:java/net/URLConnection.java':[
1261,
],
'jdk6:java/rmi/activation/ActivationGroup.java':[
285,
],
'jdk6:java/security/UnresolvedPermission.java':[
237,
241,
253,
],
'jdk6:java/util/logging/LogManager.java':[
163,
360,
740,
],
'jdk6:java/util/prefs/XmlSupport.java':[
247,
],
'jdk6:java/util/zip/ZipInputStream.java':[
442,
],
}
