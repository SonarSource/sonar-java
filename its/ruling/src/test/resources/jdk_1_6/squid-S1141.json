{
'jdk_1_6:java/awt/EventDispatchThread.java':[
374,
],
'jdk_1_6:java/awt/Toolkit.java':[
833,
838,
843,
846,
],
'jdk_1_6:java/awt/font/StyledParagraph.java':[
418,
],
'jdk_1_6:java/beans/SimpleBeanInfo.java':[
110,
],
'jdk_1_6:java/net/Inet6Address.java':[
405,
],
'jdk_1_6:java/net/InetAddress.java':[
873,
1356,
],
'jdk_1_6:java/net/PlainSocketImpl.java':[
145,
],
'jdk_1_6:java/net/URL.java':[
1139,
],
'jdk_1_6:java/net/URLClassLoader.java':[
196,
],
'jdk_1_6:java/net/URLConnection.java':[
1261,
],
'jdk_1_6:java/rmi/activation/ActivationGroup.java':[
285,
],
'jdk_1_6:java/security/UnresolvedPermission.java':[
237,
241,
253,
],
'jdk_1_6:java/util/logging/LogManager.java':[
163,
360,
740,
],
'jdk_1_6:java/util/prefs/FileSystemPreferences.java':[
558,
602,
],
'jdk_1_6:java/util/prefs/XmlSupport.java':[
247,
],
'jdk_1_6:java/util/zip/ZipInputStream.java':[
442,
],
}
