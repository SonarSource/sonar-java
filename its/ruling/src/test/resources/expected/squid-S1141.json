{
'project:com/sun/corba/se/impl/activation/ServerMain.java':[
225,
],
'project:com/sun/corba/se/impl/activation/ServerTool.java':[
386,
585,
639,
654,
738,
],
'project:com/sun/corba/se/impl/corba/RequestImpl.java':[
307,
],
'project:com/sun/corba/se/impl/encoding/CDRInputStream_1_0.java':[
1380,
2155,
2162,
],
'project:com/sun/corba/se/impl/encoding/CDROutputStream_1_0.java':[
1567,
],
'project:com/sun/corba/se/impl/interceptors/RequestInfoImpl.java':[
473,
],
'project:com/sun/corba/se/impl/io/FVDCodeBaseImpl.java':[
126,
],
'project:com/sun/corba/se/impl/io/IIOPInputStream.java':[
2089,
2282,
2334,
2345,
2463,
2475,
],
'project:com/sun/corba/se/impl/io/ObjectStreamClass.java':[
1588,
],
'project:com/sun/corba/se/impl/javax/rmi/CORBA/Util.java':[
152,
710,
],
'project:com/sun/corba/se/impl/javax/rmi/PortableRemoteObject.java':[
334,
],
'project:com/sun/corba/se/impl/naming/cosnaming/TransientNameServer.java':[
124,
],
'project:com/sun/corba/se/impl/naming/pcosnaming/NamingContextImpl.java':[
888,
961,
1001,
],
'project:com/sun/corba/se/impl/oa/poa/POAImpl.java':[
686,
],
'project:com/sun/corba/se/impl/orb/ORBConfiguratorImpl.java':[
385,
],
'project:com/sun/corba/se/impl/orb/ORBImpl.java':[
373,
],
'project:com/sun/corba/se/impl/orbutil/ObjectStreamClassUtil_1_3.java':[
506,
],
'project:com/sun/corba/se/impl/orbutil/RepositoryId_1_3.java':[
756,
896,
],
'project:com/sun/corba/se/impl/orbutil/RepositoryId_1_3_1.java':[
831,
1013,
],
'project:com/sun/corba/se/impl/orbutil/threadpool/ThreadPoolImpl.java':[
416,
],
'project:com/sun/corba/se/impl/presentation/rmi/StubConnectImpl.java':[
69,
84,
],
'project:com/sun/corba/se/impl/presentation/rmi/StubFactoryFactoryStaticImpl.java':[
127,
],
'project:com/sun/corba/se/impl/presentation/rmi/StubInvocationHandlerImpl.java':[
141,
],
'project:com/sun/corba/se/impl/protocol/CorbaClientDelegateImpl.java':[
183,
],
'project:com/sun/corba/se/impl/protocol/CorbaMessageMediatorImpl.java':[
1557,
1582,
2252,
],
'project:com/sun/corba/se/impl/protocol/POALocalCRDImpl.java':[
112,
],
'project:com/sun/corba/se/impl/transport/SelectorImpl.java':[
272,
305,
],
'project:com/sun/corba/se/impl/transport/SocketOrChannelConnectionImpl.java':[
778,
],
'project:com/sun/corba/se/impl/util/RepositoryId.java':[
831,
],
'project:com/sun/corba/se/impl/util/Utility.java':[
279,
761,
],
'project:com/sun/corba/se/spi/servicecontext/ServiceContextData.java':[
70,
81,
100,
],
'project:com/sun/jmx/remote/security/JMXPluggableAuthenticator.java':[
117,
],
'project:com/sun/jmx/snmp/SnmpEngineId.java':[
266,
273,
288,
301,
314,
],
'project:com/sun/jmx/snmp/SnmpOid.java':[
103,
],
'project:com/sun/jmx/snmp/agent/SnmpMibGroup.java':[
431,
458,
],
'project:com/sun/jmx/snmp/agent/SnmpMibTable.java':[
2119,
],
'project:com/sun/jmx/snmp/daemon/CommunicatorServer.java':[
776,
783,
],
'project:com/sun/jmx/snmp/daemon/SnmpRequestHandler.java':[
311,
],
'project:com/sun/org/apache/xalan/internal/xslt/EnvironmentCheck.java':[
448,
1065,
1079,
],
'project:com/sun/org/apache/xalan/internal/xslt/Process.java':[
763,
896,
943,
952,
995,
1002,
1020,
1029,
1111,
],
'project:com/sun/org/apache/xalan/internal/xsltc/cmdline/Transform.java':[
107,
251,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/FunctionCall.java':[
357,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Parser.java':[
455,
461,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/DocumentCache.java':[
180,
],
'project:com/sun/org/apache/xalan/internal/xsltc/runtime/BasisLibrary.java':[
1412,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/SAX2StAXEventWriter.java':[
139,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/SAX2StAXStreamWriter.java':[
106,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TrAXFilter.java':[
78,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TransformerFactoryImpl.java':[
550,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TransformerImpl.java':[
437,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/Util.java':[
93,
105,
108,
117,
],
'project:com/sun/org/apache/xerces/internal/dom/DOMMessageFormatter.java':[
79,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLEntityManager.java':[
1906,
2032,
],
'project:com/sun/org/apache/xerces/internal/impl/msg/XMLMessageFormatter.java':[
88,
],
'project:com/sun/org/apache/xerces/internal/impl/msg/XMLMessageFormatter_de.java':[
88,
],
'project:com/sun/org/apache/xerces/internal/impl/msg/XMLMessageFormatter_es.java':[
88,
],
'project:com/sun/org/apache/xerces/internal/impl/msg/XMLMessageFormatter_fr.java':[
88,
],
'project:com/sun/org/apache/xerces/internal/impl/msg/XMLMessageFormatter_it.java':[
88,
],
'project:com/sun/org/apache/xerces/internal/impl/msg/XMLMessageFormatter_ja.java':[
88,
],
'project:com/sun/org/apache/xerces/internal/impl/msg/XMLMessageFormatter_ko.java':[
88,
],
'project:com/sun/org/apache/xerces/internal/impl/msg/XMLMessageFormatter_pt_BR.java':[
88,
],
'project:com/sun/org/apache/xerces/internal/impl/msg/XMLMessageFormatter_sv.java':[
88,
],
'project:com/sun/org/apache/xerces/internal/impl/msg/XMLMessageFormatter_zh_CN.java':[
88,
],
'project:com/sun/org/apache/xerces/internal/impl/msg/XMLMessageFormatter_zh_TW.java':[
88,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XMLSchemaValidator.java':[
2433,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDHandler.java':[
2207,
2213,
2221,
2237,
2254,
],
'project:com/sun/org/apache/xerces/internal/jaxp/validation/JAXPValidationMessageFormatter.java':[
69,
],
'project:com/sun/org/apache/xerces/internal/jaxp/validation/ValidatorHandlerImpl.java':[
680,
],
'project:com/sun/org/apache/xerces/internal/parsers/AbstractSAXParser.java':[
1875,
1893,
],
'project:com/sun/org/apache/xerces/internal/util/DatatypeMessageFormatter.java':[
71,
],
'project:com/sun/org/apache/xerces/internal/util/SAXMessageFormatter.java':[
69,
],
'project:com/sun/org/apache/xerces/internal/util/XMLCatalogResolver.java':[
348,
],
'project:com/sun/org/apache/xml/internal/resolver/CatalogManager.java':[
681,
],
'project:com/sun/org/apache/xml/internal/resolver/readers/TR9401CatalogReader.java':[
107,
],
'project:com/sun/org/apache/xml/internal/resolver/readers/TextCatalogReader.java':[
146,
],
'project:com/sun/org/apache/xml/internal/security/Init.java':[
135,
192,
230,
273,
323,
],
'project:com/sun/org/apache/xml/internal/security/signature/Manifest.java':[
323,
337,
],
'project:com/sun/org/apache/xml/internal/security/signature/XMLSignature.java':[
511,
616,
],
'project:com/sun/org/apache/xml/internal/security/transforms/implementations/TransformBase64Decode.java':[
147,
],
'project:com/sun/org/apache/xml/internal/security/transforms/implementations/TransformXSLT.java':[
164,
],
'project:com/sun/org/apache/xml/internal/serializer/CharInfo.java':[
215,
249,
],
'project:com/sun/org/apache/xml/internal/serializer/Encodings.java':[
314,
],
'project:com/sun/org/apache/xml/internal/utils/DOMHelper.java':[
95,
],
'project:com/sun/org/apache/xml/internal/utils/Hashtree2Node.java':[
107,
],
'project:com/sun/org/apache/xml/internal/utils/XMLReaderManager.java':[
106,
114,
127,
],
'project:com/sun/org/apache/xpath/internal/SourceTreeManager.java':[
354,
374,
],
'project:com/sun/security/auth/PolicyFile.java':[
349,
597,
],
'project:java/awt/GraphicsEnvironment.java':[
99,
],
'project:java/awt/Toolkit.java':[
869,
874,
],
'project:java/awt/font/StyledParagraph.java':[
443,
],
'project:java/beans/SimpleBeanInfo.java':[
128,
],
'project:java/io/Console.java':[
524,
],
'project:java/net/CookieManager.java':[
283,
],
'project:java/net/Inet6Address.java':[
427,
],
'project:java/net/InetAddress.java':[
886,
1433,
],
'project:java/net/URL.java':[
1164,
],
'project:java/net/URLClassLoader.java':[
360,
],
'project:java/net/URLConnection.java':[
1320,
],
'project:java/nio/file/FileTreeWalker.java':[
104,
108,
],
'project:java/rmi/activation/ActivationGroup.java':[
302,
],
'project:java/security/UnresolvedPermission.java':[
253,
257,
269,
],
'project:java/util/logging/LogManager.java':[
181,
420,
784,
],
'project:java/util/prefs/FileSystemPreferences.java':[
572,
617,
],
'project:java/util/prefs/XmlSupport.java':[
264,
],
'project:javax/imageio/metadata/IIOMetadata.java':[
413,
427,
],
'project:javax/management/MBeanServerFactory.java':[
482,
],
'project:javax/management/modelmbean/RequiredModelMBean.java':[
1592,
],
'project:javax/management/remote/rmi/RMIConnectionImpl.java':[
1379,
1542,
],
'project:javax/management/remote/rmi/RMIConnector.java':[
295,
],
'project:javax/management/remote/rmi/RMIConnectorServer.java':[
422,
],
'project:javax/print/PrintServiceLookup.java':[
346,
377,
],
'project:javax/print/StreamPrintServiceFactory.java':[
184,
],
'project:javax/script/ScriptEngineManager.java':[
126,
],
'project:javax/sound/sampled/AudioSystem.java':[
243,
],
'project:javax/swing/JEditorPane.java':[
635,
],
'project:javax/swing/TimerQueue.java':[
170,
173,
],
'project:javax/swing/text/DefaultCaret.java':[
450,
],
'project:javax/xml/bind/ContextFinder.java':[
169,
],
'project:org/w3c/dom/bootstrap/DOMImplementationRegistry.java':[
307,
],
}
